src/subscript/co2containment/co2_containment/calculate.py

```diff
@@ -20,11 +21,23 @@ def __post_init__(self):
             d = self.date
             self.date = f"{d[:4]}-{d[4:6]}-{d[6:]}"
 
+@dataclass
+class ContainedCo2Vol:
+    date: str
+    amount_m3: float
+    phase: Literal["gas", "aqueous","extent"]
```

Comment:
I think "extent" is better placed in a new variable, since phase is a bit misleading? Maybe a variable "volume_type" that could be "extent" (or "simplistic" or ?....) or "representative" (or "actual") or "gas_phase_approximation".